code/renderers/vue3/src/render.ts

```diff
@@ -54,7 +65,7 @@ export function renderToDOM(
 
   showMain();
 
-  map.get(domElement)?.unmount();
+  componentsByDomElementId.get(domElementKey)?.unmount();
```

Comment:
DOM elements are actually not reference equal in docs mode, so this would actually never cleanup and unmount those component, keeping them in memory forever.

Reply:
> DOM elements are actually not reference equal in docs mode

Why is that?

Reply:
Should we use a teardown for this instead so we can maintain the element=>app map?